tests/js/spec/views/organizationStatsV2/index.spec.jsx

```diff
@@ -6,10 +6,10 @@ import {initializeOrg} from 'sentry-test/initializeOrg';
 
 import {DEFAULT_RELATIVE_PERIODS, DEFAULT_STATS_PERIOD} from 'app/constants';
 import {DataCategory} from 'app/types';
-import UsageStats from 'app/views/usageStats';
-import {CHART_OPTIONS_DATA_TRANSFORM} from 'app/views/usageStats/usageChart';
+import {OrganizationStats} from 'app/views/organizationStatsV2';
+import {CHART_OPTIONS_DATA_TRANSFORM} from 'app/views/organizationStatsV2/usageChart';
```

Comment:
Are you planning on renaming this to `organizationStats` once the old code is removed? Keeping a v2 around long term isn't great.

Reply:
Deleted v1 on frontend, so there won't be v1/v2 there. 

The backend will still have v1/v2 as there are a non-trivial amount of orgs hitting the v1 endpoint.